portfolio/src/main/webapp/script.js

```diff
@@ -62,3 +62,67 @@ function changeProfilePic() {
   document.getElementById('pfp-link').href = '/images/' + img;
   document.getElementById('pfp').src = '/images/' + img;
 }
+
+let xando = [
+  [0, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0],
+];
+
+/**
+ * Process tic-tac-toe game.
+ */
+function ticTacToe(cell) {
+  const playerTurn = document.getElementById('player-turn').className;
+  const whichCell = document.getElementById(cell);
+  if (playerTurn === '0') {
+    whichCell.innerText = '❌';
+  } else {
+    whichCell.innerText = '⭕';
+  }
+  whichCell.disabled = true;
+
+  if (cell < 4) {
+    xando[+playerTurn][0]++;
+  } else if (cell < 7) {
+    xando[+playerTurn][1]++;
+  } else {
+    xando[+playerTurn][2]++;
+  }
+
+  if (cell === 1 || cell === 4 || cell === 7) {
+    xando[+playerTurn][3]++;
+  } else if (cell === 2 || cell === 5 || cell === 8) {
+    xando[+playerTurn][4]++;
+  } else {
+    xando[+playerTurn][5]++;
+  }
+
+  if (cell === 1 || cell === 5 || cell === 9) {
+    xando[+playerTurn][6]++;
+  } else if (cell === 3 || cell === 5 || cell === 7) {
+    xando[+playerTurn][7]++;
+  }
+
+  let winner;
+
+  for (let i = 0; i < 2; i++) {
```

Comment:
Might as well use a meaningful variable name for this:

for (let player = 0; player < 2; player++) {

Reply:
Done